README.md

```diff
@@ -29,7 +29,7 @@
 ```
 .
 ├── proxies
-│   └── events-federation-gateway (Apollo Router and Gateway)
+│   └── events-graphql-router (Apollo Router and Gateway)
```

Comment:
I think `federation` could be a bit better name for the whole thing instead of `router`, because it would show right away that it is precisely Apollo Federation. When outsiders/newcomers look at the code and the configs, it might take some time to figure out that `router` actually means Apollo Federation router and not something else.

Reply:
so,
1. events-graphql-federation?
2. events-federation-router?

I chose the router "events-graphql-router", because 
- "events" is the scope that this supergraph has
- "graphql" is the used protocol, no matter which client is being used: apollo, or somethign else
- "router" because the Apollo talks about router when they talk about this new Rust runtime executable. 

The folder contains nothing but a 
1. supergraph that the router routes
2. configuration for the router
so, all what the router needs. Okay, it's true that managing the supergraph is not that much related to the router, but to the federation.

But now you @tuomas777 need to collaborate with the new name! 😛

Reply:
Okay - it was decided with @tuomas777 to rename the service to `events-graphql-federation`. So, I changed the 

1. the directory name of the service's directory
2. the Apollo client name in both the apps (hobbies & events)
3. the environment variables related to the Apollo Router

And I also updated the READMEs